src/pages/components/CharacterLimit/CharacterLimit.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+
+import { ICharacterLimitProps } from './CharacterLimit.types';
+
+const CharacterCountdown: React.FC<ICharacterLimitProps> = (props) => {
```

Comment:
Vamos seguir o styleguide e todo componente deve ser function.

```suggestion
function CharacterCountdown(props): React.FC<ICharacterLimitProps>{
```